lib/interfaces/common.js

```diff
@@ -109,7 +110,10 @@ module.exports = function (suites, context, mocha) {
           suite.parent._onlySuites = suite.parent._onlySuites.concat(suite);
         }
         if (typeof opts.fn === 'function') {
-          opts.fn.call(suite);
+          var result = opts.fn.call(suite);
+          if (result) {
```

Comment:
I believe that keeping this `typeof result === 'undefined'` is more consistent since we are deprecating all return values.

Reply:
Original code was `if (typeof result !== 'undefined') {` and I asked to simplify it.
I believe that `if (result)` is enough even if we are deprecating all return values.
What do you think?

Reply:
@outsideris I'd vote for consistency. Also i find `typeof ... 'undefined'` simple for all JS developers IMO

Reply:
My vote's for `typeof ... 'undefined'` as well; this codebase is not 100% consistent in this matter but I would agree that it's considered idiomatic (if quirky) javascript.

Reply:
@jleedev Sorry for making to fix it twice.